kalite/distributed/static/js/distributed/video/hbtemplates/video-player.handlebars

```diff
@@ -1,9 +1,11 @@
 <div id="video-player"{{#unless content_urls }} class="client-online-only"{{/unless}}>
     <video class="video-player video-js vjs-default-skin vjs-fullscreen" id="{{ random_id }}">
         <source src="{{ content_urls.stream }}" type="{{ content_urls.stream_type }}" />
-        {{#each subtitle_urls }}
-        <track kind="captions" src="{{ this.url }}" srclang="{{ code }}" label="{{ name }}" {{#ifcond code "==" ../selected_language }}{{#ifcond ../code "!=" "en" }}default="True"{{/ifcond}}{{/ifcond}}/>
-        {{/each}}
+        {{#current_language_is "en"}}
+            {{#each subtitle_urls }}
+                <track kind="captions" src="{{ this.url }}" srclang="{{ this.code }}" label="{{ this.name }}" {{#ifcond this.code "==" "en"}}default{{/ifcond}}/>
```

Comment:
Looking at the source, I have a few questions:

1.  Out of curiosity, why did you use `this` at `srclang="{{ this.code }}"...` instead of `srclang="{{ code }}"...`?
2.  With the omitted lines, we make the selected language (that is not English) the default language on the list.  But with your updates, we only set English as the default if it's the current language.  Pardon the memory lapse, but are we certain that we are playing a dubbed video if current language is not English?